server/src/main/kotlin/suwayomi/tachidesk/manga/model/dataclass/ChapterDataClass.kt

```diff
@@ -8,6 +8,7 @@ package suwayomi.tachidesk.manga.model.dataclass
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 data class ChapterDataClass(
+    val id: Int,
```

Comment:
Why are we exposing chapter id here? 
`XxxxDataClass` is the representation of an object understandable by the api clients.
For client side work we usually use `XxxTable` and `ResultRow`

Reply:
> Since this is a new API we could add id to chapters in all the endpoints and have this new one take chapter ID. All new batch API endpoints could then use chapter ids.

If we want to do this then we should simplify api endpoints all over and declare a breaking change. Otherwise it'll not be coherent.

Reply:
https://github.com/Suwayomi/Tachidesk-Server/pull/436#discussion_r1014744951
Its hard to grab specific chapters from the database using a list of manga ids and chapter indexes

Reply:
> Its hard to grab specific chapters from the database using a list of manga ids and chapter indexes

I knowledge that, but private methods don't have to use data class objects

Reply:
@AriaMoradi It is not practical to work with combination of identifiers because Exposed can't natively perform selects with such identifiers (combined keys).

This also simplify API. Client can explicitly identify chapters it wants to execute actions on, instead of using some combinations of related object id and index number. It does not introduce any added complexity, since clients already have to fetch chapters to use theirs index for identification.

Since chapter id alone is enough to identify chapter, there could be new endpoints introduced which would not require mangaId (ie. `/api/v1/chapter/:chapterId`). Clients could migrate to these over time and then breaking change can deprecate old API. (If this is wanted at some point in the future...)

> If we want to do this then we should simplify api endpoints all over and declare a breaking change. Otherwise it'll not be coherent.

This would block this PR for a while and later break all the clients. Why do that if this can be non-breaking change?

Reply:
Alright, I accept your argument.